bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/EntryLocationIndex.java

```diff
@@ -73,18 +74,28 @@ public long getLocation(long ledgerId, long entryId) throws IOException {
         LongPairWrapper key = LongPairWrapper.get(ledgerId, entryId);
         LongWrapper value = LongWrapper.get();
 
+        long startTimeNanos = MathUtils.nowInNano();
+        boolean operationSuccess = false;
         try {
             if (locationsDb.get(key.array, value.array) < 0) {
                 if (log.isDebugEnabled()) {
                     log.debug("Entry not found {}@{} in db index", ledgerId, entryId);
                 }
                 return 0;
             }
-
+            operationSuccess = true;
             return value.getValue();
         } finally {
             key.recycle();
             value.recycle();
+            long eventLatencyMillis = MathUtils.elapsedNanos(startTimeNanos);
```

Comment:
Nit
```suggestion
            long eventLatencyNanos = MathUtils.elapsedNanos(startTimeNanos);
```

Reply:
Thanks for catching that. I'll inline it as suggested by @StevenLuMT .

Reply:
@michaeljmarshall  I now noticed where the name `eventLatencyMillis` came from. This was from the OpStatsLogger interface methods where the parameter was named `eventLatencyMillis`. When I extracted the variable with IntelliJ extract variable refactoring, it suggested `eventLatencyMillis` as the variable name. I'm fixing that inconsistency as part of this PR.